### ISSUE TITLE
Flaky test_symbol_list_parallel_stress_with_delete test

### ISSUE THREAD

**Issue by IvoDD** [man-group/ArcticDB#1195]
### Describe the bug

The test_symbol_list_parallel_stress_with_delete test does a bunch of writes in parallel and we can sometimes end up in an invalid state.

See the failure: https://github.com/man-group/ArcticDB/actions/runs/7384650532/job/20088433429?pr=1181

### Steps/Code to Reproduce

Rerun the test.

### Expected Results

The test always passes

### OS, Python Version and ArcticDB Version

Python 3.7: https://github.com/man-group/ArcticDB/actions/runs/7384650532/job/20088433429?pr=1181

**Comment by IvoDD** [comment]
Duplicate of https://github.com/man-group/ArcticDB/issues/1198
